specification/cognitiveservices/data-plane/Personalizer/readme.md

```diff
@@ -0,0 +1,227 @@
+# Cognitive Serves Personalizer C# SDK quickstart samples 
```

Comment:
Please create a readme.md file similar to [this](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cognitiveservices/data-plane/ComputerVision/readme.md). These C# specific instructions can be moved to another readme.md file in the https://github.com/Azure/azure-sdk-for-net repo

Reply:
fixed